src/introduction/qemu.md

```diff
@@ -0,0 +1,33 @@
+# Running QEMU
+
+This section is optional; some users of this guide will have been provided
+environments for building and running CHERI systems and programs.  If you have
```

Comment:
Weird way to phrase it when people have acquired Morello boards (not really "provided environments", they installed it on their hardware).